### ISSUE TITLE
Links in AUR (Arch User Repository) emails are no longer clickable

### ISSUE THREAD

**Issue by yochananmarqos** [tutao/tutanota#3140]
- [x ] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [ x] I've searched and did not find a similar issue.

**Bug in web / Linux / Android app**

**Describe the bug**
This happens on the web (3.84.0), on Linux (3.84.0) and Android (3.84.1). The links were clickable with <=3.83.7.

Emails from notify@aur.archlinux.org contain links to the AUR page that appear as such:

```
[1] https://aur.archlinux.org/account/username/
[2] https://aur.archlinux.org/pkgbase/package-name/
```

Also from Arch Linux Forums Mailer <noreply@archlinux.org> with the forum links:

```
The post is located at https://bbs.archlinux.org/viewtopic.php?pid=123456
```

```
You can unsubscribe by going to https://bbs.archlinux.org/viewtopic.php?id=123456#unsubscribe and clicking the Unsubscribe link at the bottom of the page.
```

**Desktop (please complete the following information):**
 - OS: Manjaro GNOME
 - Browser: Google Chrome 91.0.4472.77
 - Version: Tutanota Desktop 3.84.0

**Smartphone (please complete the following information):**
 - Device: OnePlus 7 Pro
 - OS: Android 11
 - Browser: N/A
 - Version: Tutanota Desktop 3.84.1

**Comment by charlag** [comment]
Linking functionality might have been affected, we are onto it.
I wanted to point out that those links were newer formatted as links

**Comment by yochananmarqos** [comment]
Yes, it was affected as my issue describes. I'm not sure how Tutanota did or did not format the links, but they were clickable before updating Tutanota to the aformentioned affected versions.

**Comment by ark-** [comment]
I'm also getting this on a plaintext email with a URL in it. I used to be able to click the link which was extremely convenient. 

Now I'm back to using fiddly touch controls to copy and paste into my browser.

**Comment by charlag** [comment]
This was fixed with #3192
